tsl/test/expected/dist_policy.out

```diff
@@ -0,0 +1,163 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+\unset ECHO
+psql:include/remote_exec.sql:5: NOTICE:  schema "test" already exists, skipping
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
+RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_auto_drop_chunks'
+LANGUAGE C VOLATILE STRICT;
+-- Add data nodes
+\x on
+SELECT * FROM add_data_node('dist_policy_data_node_1', host => 'localhost',
+                            database => 'dist_policy_data_node_1');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_1
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_1
+node_created      | t
+database_created  | t
+extension_created | t
+
+SELECT * FROM add_data_node('dist_policy_data_node_2', host => 'localhost',
+                            database => 'dist_policy_data_node_2');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_2
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_2
+node_created      | t
+database_created  | t
+extension_created | t
+
+SELECT * FROM add_data_node('dist_policy_data_node_3', host => 'localhost',
+                            database => 'dist_policy_data_node_3');
+-[ RECORD 1 ]-----+------------------------
+node_name         | dist_policy_data_node_3
+host              | localhost
+port              | 55432
+database          | dist_policy_data_node_3
+node_created      | t
+database_created  | t
+extension_created | t
+
+\x off
+GRANT USAGE ON FOREIGN SERVER dist_policy_data_node_1, dist_policy_data_node_2, dist_policy_data_node_3 TO PUBLIC;
+SET ROLE :ROLE_1;
+CREATE TABLE conditions(
+    time timestamptz NOT NULL, 
+    device int, 
+    value float
+);
+SELECT * FROM create_distributed_hypertable('conditions', 'time', 'device', 3,
+       chunk_time_interval => INTERVAL '1 week');
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             1 | public      | conditions | t
+(1 row)
+
+INSERT INTO conditions
+SELECT time, device, random()*80
+FROM generate_series(NOW() - INTERVAL '4 weeks', NOW(), '1 week') AS time,
```

Comment:
using now like that will produce flaky test

Reply:
yes, this can produce different number of chunks and different chunk names depending on when you run the test.

Reply:
It is used in a number of other tests. In addition, I do not understand how it can produce a flaky test. It creates one row for each chunk spaced exactly 1 week apart, which is the same as the chunk interval size. This should be able to place more than one row in a chunk.

Reply:
Switched to integer-based time.